crates/spk-cli/group4/src/cmd_lint.rs

```diff
@@ -25,12 +27,26 @@ impl Run for Lint {
     type Output = i32;
 
     async fn run(&mut self) -> Result<Self::Output> {
-        let options = self.options.get_options()?;
         let mut out = 0;
+        let options = self.options.get_options()?;
         for spec in self.packages.iter() {
-            let result = SpecTemplate::from_file(spec).and_then(|t| t.render(&options));
-            match result {
-                Ok(_) => println!("{} {}", "OK".green(), spec.display()),
+            let yaml = SpecTemplate::from_file(spec).and_then(|t| t.render_to_string(&options))?;
+            let lints: std::result::Result<LintedItem<Spec<AnyIdent>>, serde_yaml::Error> =
```

Comment:
I think it is a problem if this is hard coded to expect `spk_schema::v0::Spec` here, rather than the more general `spk_schema::Spec`. The lint command needs to work for all spec types/versions.